README.md

```diff
@@ -26,12 +39,14 @@ If you want to display algs or solves, the following code is a good way to get s
 <twisty-player alg="R U R' U R U2' R'"></twisty-player>
 ```
 
-You can find more documentation at [`js.cubing.net/cubing/`](https://js.cubing.net/cubing).  
-The source for the documentation site is currently inside the [`docs`](./docs/) folder of this repository.
+You can find more documentation at [js.cubing.net/cubing/](https://js.cubing.net/cubing).  
+The source for the documentation site is currently inside the [docs](./docs/) folder of this repository.
+
+## Contributing
 
-## Development
+If you would like to contribute to the development of `cubing.js`, please refer to our [contribution guidelines](CONTRIBUTING.md).
 
-Development relies on [`node`](https://nodejs.org/en/) and [`npm`](https://docs.npmjs.com/getting-started). Once you have installed those, you can run:
+Development relies on [node](https://nodejs.org/en/) and [npm](https://docs.npmjs.com/getting-started) (installing `node` will install `npm` as well). Once you have installed those, you can run:
```

Comment:
Hmm, the `npm` docs here aren't really that great. I should probably have left them out. How's this?

```suggestion
To work on the `cubing.js` source code, you'll need to install [`node` (and `npm`)](https://nodejs.org/en/). Once you have installed those, you can run:
```

Reply:
I agree the npm docs suck xD From a beginner perspective, I like the inclusion of "(installing `node` will install `npm` as well)", as that took me a second to figure out, and seems to be such ingrained knowledge in the community that it took me a surprisingly amount of Googling to confirm that.

Reply:
Gonna change this to the middle:

```suggestion
To work on the `cubing.js` source code, you'll need to install [`node`](https://nodejs.org/en/) and [npm](https://docs.npmjs.com/getting-started) (installing `node` will install `npm` as well). Once you have installed those, you can run:
```